harness/determined/common/storage/s3.py

```diff
@@ -8,6 +8,7 @@
 
 from determined import errors
 from determined.common import storage, util
+from determined.errors import NoDirectStorageAccess
```

Comment:
outside of the CLI, which has never been reigned in, we follow the [google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports), which basically says "other than `typing` don't import symbols directly".

You'll notice there's already a `from determined.errors` in this file, so you can delete this line and use `errors.NoDirectStorageAccess`.  The import patterns with that go style guide are a lot like go: you import the package and then use `package.Symbol`.